manual/src/SUMMARY.md

```diff
@@ -0,0 +1,18 @@
+# Summary
+
+- [Introduction](README.md)
+
+# Basic Usage
+
+- [Installation](basic/installation.md)
+- [Feature Choices](basic/feature_choices.md)
+- [Connection and Operation](./basic/connection_and_operation.md)
+
+# Advanced
+
```

Comment:
I think some more good sections to have would be "Cursors", "Sessions and Transactions", "Change Streams", and "Monitoring" (for stuff like command, CMAP, and SDAM monitoring). Perhaps, we'd want "Reading from the Database" and "Writing to the Database" sections too, and "Cursors" could just be a subsection of the former.

Reply:
Done.